src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -1948,7 +1949,13 @@ private FeedbackSessionResultsBundle getFeedbackSessionResultsForUserWithParams(
                         }
                         isVisibleResponse = false;
                     }
-
+                }
+                
+                if (question.questionType == FeedbackQuestionType.CONTRIB) {
```

Comment:
Regarding this, this section of code will only apply in question view. Since question view loads each question asynchronously, with each question individually retrieving a bundle, each bundle will only contain data structures with data for a single type of question. Thus this section of code only returns a bundle which contains data for one question.

I might have misunderstood what you meant, could you give me an example of a possible issue that might arise?